docs/section1/Rule 1-6.md

```diff
@@ -0,0 +1,36 @@
+
+# Section 1 - Rule 1-6 
+
+**Rule ID:** 1-6   
+**Rule Description:** On-site renewable energy shall not be included in the baseline building performance.  
+**Rule Assertion:** Baseline RMD = expected value  
+**Appendix G Section:** G3.11 18 Baseline 
+
+**Mandatory Rule:** True    
+**Evaluation Context:** Each RMD  
+**Function Call:**  
+
+## Applicability Check:  
+- All projects are applicable
+
+
+## Rule Logic:
+- get the baseline output schema: `output = B_RMD.output`
+- get the output instance: `output_instance = output.output_instance`
+- set a boolean has_renewables and set it to false: `has_renewables = false`
+- look at each end use result: `for end_use_result in output_instance.annual_end_use_results:`
+    - check if the energy source for the end_use_result is "ON_SITE_RENEWABLES": `if end_use_result.energy_source == "ON_SITE_RENEWABLES":`
+        - set has_renewables to true and continue to rule assertion: `has_renewables = true; CONTINUE TO RULE ASSERTION`

```

Comment:
maybe we could also check if the annual_site_energy_use >0 when the energy source = on site renewables

Reply:
changed